<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dojizdeni</class>
 <widget class="QWidget" name="dojizdeni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radPupik">
     <property name="text">
      <string>Jindra</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radAutem">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="radJinak">
     <property name="text">
      <string>Jinak</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radKolo">
     <property name="text">
      <string>Kolo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labPlatit">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
